libs/ui/src/components/atoms/Tooltip/Tooltip.styles.ts

```diff
@@ -9,7 +9,7 @@ import { border } from '../../../theme/global/border';
 export const TooltipProvider = TooltipPrimitive.Provider;
 export const TooltipRoot = TooltipPrimitive.Root;
 export const TooltipTrigger = styled(TooltipPrimitive.Trigger)`
-  background-color: ${({ theme }: { theme: Theme }) => theme.bgColor};
+  background-color: transparent;
```

Comment:
just checking - was this suppose to be changed?

Reply:
Yeah, the background was colored with the primary theme. It made it so there was a small box that was a different color than the card. It makes more sense to have the background be the color of the parent.